administrator/language/de-DE/com_users.ini

```
@@ -38,7 +38,7 @@ COM_USERS_CONFIG_FIELD_ENFORCE_2FA_FIELD_LABEL="Zwei-Faktor-Authentifizierung er
 COM_USERS_CONFIG_FIELD_ENFORCE_2FA_FIELD_SITE="Seite (Frontend)"
 COM_USERS_CONFIG_FIELD_FRONTEND_LANG_LABEL="Frontend-Sprache"
 COM_USERS_CONFIG_FIELD_FRONTEND_RESET_COUNT_LABEL="Zurücksetzungsmaximum"
-COM_USERS_CONFIG_FIELD_FRONTEND_RESET_TIME_DESC="Die Zeitspanne (in Stunden) in der die maximalen Passwortzurücksetzungen erlaubt sind."
+COM_USERS_CONFIG_FIELD_FRONTEND_RESET_TIME_LABEL="Zurücksetzungs-Zeit"
```

```suggestion
COM_USERS_CONFIG_FIELD_FRONTEND_RESET_TIME_LABEL="Zurücksetzungsdauer"
```
Umgangssprachlich und mehr Treffer in Google.